dmoj/executors/BF.py

```diff
@@ -185,15 +185,15 @@ def compile_to_llvm(source_code: bytes) -> bytes:
 
 OPT_PASSES = [
     # Shorten instructions
-    'instcombine',
+    'instcombine<no-verify-fixpoint>',  # https://github.com/llvm/llvm-project/issues/92648
```

Comment:
We know the llvm version because we use it for reporting. Can we just detect it here?

Reply:
done

opt 17.0.1 works with `instcombine` and opt 18.1.0 works with `instcombine<no-verify-fixpoint>`, so i set the cutoff to 18